### ISSUE TITLE
About loss function used during training of Fixer

### ISSUE THREAD

**Issue by LeeKeyu** [nv-tlabs/Fixer#8]
Dear authors, thanks a lot for sharing your work Fixer!!! My question is about the loss function used. It seems that GAN, clipsim and gram loss were included in the code but the implementation of the loss funcitions and different training stages are missing. Did you use these losses during training of Fixer model? Were they helpful than only using L2 and LPIPIS losses?
Thanks so much for your help in advance!
